src/game/SpellMgr.cpp

```diff
@@ -26,6 +26,8 @@
 #include "Spell.h"
 #include "Unit.h"
 #include "World.h"
+#include "SpellAuras.h"
+
```

Comment:
There's no reason for this empty line when you have one just on the next line =)